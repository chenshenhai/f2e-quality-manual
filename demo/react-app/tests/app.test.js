import App from './../src/app';

import React from 'react';
import renderer from 'react-test-renderer';

test('Link changes the class when hovered', () => {
  const component = renderer.create(
    <App title={'hello world'} info={'this is a react-app'} />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});