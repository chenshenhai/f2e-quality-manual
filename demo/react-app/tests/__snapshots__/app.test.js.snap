// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<div>
  <h1>
    hello world
  </h1>
  <p>
    this is a react-app
  </p>
</div>
`;
